content/TrueCommand/Alerts/AlertManage.md

```diff
@@ -6,125 +6,179 @@ aliases:
+{{< tabs "Alert Screens" >}}
+{{< tab "All Alerts" >}}
 
-The **Active Alerts** tab shows all unresolved alerts.
-Alerts are moved to the **Resolved Alerts** tab by clicking *Resolve* Selected <i class="material-icons" aria-hidden="true" title="Check">check</i>.
-To resolve multiple alerts, select each alert and click *Resolve* Selected<i class="material-icons" aria-hidden="true" title="Check">check</i>.
+## Viewing Alerts
 
-Click *View alert notice details (`i`)* to view any user comments about an alert.
+To see all alerts discovered by TrueCommand, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click **All Alerts**.
+Administrator accounts can see all system alerts.
+A non-administrator account can only view alerts according to their team and user account permissions.
 
-Administrator accounts can delete an alert by clicking *Delete* Selected<i class="material-icons" aria-hidden="true" title="Delete">delete</i>.
-Deleting an alert cannot be undone.
-To delete multiple alerts, select each alert and click *Delete* Selected<i class="material-icons" aria-hidden="true" title="Delete">delete</i>.
+![AllAlerts](/images/TrueCommand/2.1/AllAlerts.png "All Alerts")
 
 ### Viewing Alerts by System
 
-Alerts generated by a monitored system display in both the administrative **Systems** screen and the **Dashboard** as a number above the system icon.
-To view all alerts for a single system, go to the **Dashboard** and select a single system.
+Alerts generated by a monitored system display on both the **Systems** screen and the system card or detail single system view on the **Dashboard** as a number to the right of the system name.
 
-![DashboardSystemAlerts](/images/TrueCommand/2.0/DashboardSystemAlerts.png "Dashboard: System Alerts")
+![DashboardSystemAlerts](/images/TrueCommand/2.0/DashboardSystemAlerts.png "Dashboard System Alerts")
 
-Each active and resolved alert for that system is visible in this tab.
-Clicking *View alert notice details (`i`)* shows details for that alert, including the option to leave comments about the alert.
+To view all alerts for a single system, from the **Dashboard** and click on a system name with alerts indicated. That system's details screen displays with the alerts listed in the **Alerts** area of the screen.
 
-## Managing Alert Rules
+![DashboardSystemDetailAlerts](/images/TrueCommand/2.1/DashboardSystemDetailAlerts.png "Dashboard System Detail Alerts")
+
+Click the <mat-icon _ngcontent-bbg-c138="" role="img" fontset="mdi" fonticon="mdi-comment-text-multiple" class="mat-icon notranslate mdi mdi-comment-text-multiple mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-comment-text-multiple" data-mat-icon-namespace="mdi"></mat-icon> icon in the **Alerts** area of the system detail screen to display the **Notice Details** side panel with information about the alert and any user comments made.
```

Comment:
This <mat-icon> code should be able to be pared down to the <span class="iconify" data-icon="mdi:comment-text-multiple"></span>
```suggestion
Click the <span class="iconify" data-icon="mdi:mdi-comment-text-multiple"></span> icon in the **Alerts** area of the system detail screen to display the **Notice Details** side panel with information about the alert and any user comments made.
```